### PR TITLE
Be explicit about newlines in Process.OutputDataReceived

### PR DESCRIPTION
**Pull request by svick**
The documentation is currently not clear about what counts as "a newline character" for the purposes of `Process.OutputDataReceived` and `Process.ErrorDataReceived` and I think it should, so this PR fixes that.

### CODE REVIEW

**Comment by carlossanlop** [comment]
Area owners: @adamsitnik @eiriktsarpalis 
Can you please review?

**Review by adamsitnik — APPROVED** [review]
LGTM, thank you @svick !

File: xml/System.Diagnostics/Process.xml [hunk 1]
```diff
@@ -1033,7 +1033,7 @@ The following code example creates a process that prints a file. It sets the <xr
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated process has written to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.  
+ The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line, terminating with a newline (CR, LF or CR+LF), to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.
```
Comment (gewarren) [inline xml/System.Diagnostics/Process.xml:1036]:
```suggestion
 The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line that's terminated with a newline (`CR`, `LF`, or `CR+LF`) to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.   
```

File: xml/System.Diagnostics/Process.xml [hunk 2]
```diff
@@ -3129,7 +3129,7 @@ The calling process is a member of the associated process' descendant tree.</exc
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Diagnostics.Process.OutputDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line, terminating with a newline character, to its redirected <xref:System.Diagnostics.Process.StandardOutput%2A> stream.  
+ The <xref:System.Diagnostics.Process.OutputDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line, terminating with a newline (CR, LF or CR+LF), to its redirected <xref:System.Diagnostics.Process.StandardOutput%2A> stream.
```
Comment (gewarren) [inline xml/System.Diagnostics/Process.xml:3132]:
```suggestion
 The <xref:System.Diagnostics.Process.OutputDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line that's terminated with a newline (`CR`, `LF`, or `CR+LF`) to its redirected <xref:System.Diagnostics.Process.StandardOutput%2A> stream.   
```

File: xml/System.Diagnostics/Process.xml (hunk 1 above)
Comment (svick) [reply xml/System.Diagnostics/Process.xml:1036]: Why should CR and LF be displayed as code? It's not code in some programming language (like e.g. `\r\n` or `vbCrLf`), they are abbreviations and I think those should be shown as regular text.

Comment (gewarren) [reply xml/System.Diagnostics/Process.xml:1036]: I made a different suggestion. Feel free to accept it or not. I would at least accept the "that's terminated" part though.

Comment (svick) [reply xml/System.Diagnostics/Process.xml]: @gewarren Yeah, that sounds good to me, accepted.

**Review by gewarren — APPROVED** [review]

File: xml/System.Diagnostics/Process.xml [hunk 3]
```diff
@@ -3129,7 +3129,7 @@ The calling process is a member of the associated process' descendant tree.</exc
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Diagnostics.Process.OutputDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line, terminating with a newline (CR, LF or CR+LF), to its redirected <xref:System.Diagnostics.Process.StandardOutput%2A> stream.  
+ The <xref:System.Diagnostics.Process.ErrorDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line that's terminated with a newline (carriage return (CR), line feed (LF), or CR+LF) to its redirected <xref:System.Diagnostics.Process.StandardError%2A> stream.
```
Comment (gewarren) [inline xml/System.Diagnostics/Process.xml:3132]:
```suggestion
 The <xref:System.Diagnostics.Process.OutputDataReceived> event indicates that the associated <xref:System.Diagnostics.Process> has written a line that's terminated with a newline (carriage return (CR), line feed (LF), or CR+LF) to its redirected <xref:System.Diagnostics.Process.StandardOutput%2A> stream.
```

**Review by gewarren — CHANGES_REQUESTED** [review]
Oops, looks like a copy paste error.

**Review by gewarren — APPROVED** [review]

File: xml/System.Diagnostics/Process.xml (hunk 3 above)
Comment (svick) [reply xml/System.Diagnostics/Process.xml]: @gewarren Thanks for fixing that for me.

**Pull request merged**